autogpt_platform/backend/backend/server/rest_api.py

```diff
@@ -32,24 +31,23 @@
 
 
 class AgentServer(AppService):
-    mutex = KeyedMutex()
-    use_redis = True
+    use_queue = True
     _test_dependency_overrides = {}
     _user_credit_model = get_user_credit_model()
 
-    def __init__(self, event_queue: AsyncEventQueue | None = None):
+    def __init__(self):
         super().__init__(port=Config().agent_server_port)
-        self.event_queue = event_queue or AsyncRedisEventQueue()
+        self.event_queue = RedisEventQueue()
```

Comment:
Since we are already forcing Redis, there is no added value of mocking the queue in our integration test.
OTOH, we need some test coverage on this.